### DISCUSSION TITLE
[Login Site] compatibility with the Authentication Service

### DISCUSSION THREAD

**Discussion by malta895** [mia-platform/community#210] [Q&A] [answered]
Hello,
I need to use the login site with a provider which is not Auth0.

In Mia-Platform docs, I only find it used in flows with the Auth0 client, and in the console I can only create it in Applications with the auth0-client, such as the Secure Backoffice.

Is there a way to set up a project with the Login Site in a way that the authentication is provided by a vendor different from Auth0, by means of the Authentication service?

**Comment by davidebianchi** [comment, marked as answer]
Hi @malta895,

yes it is possible to use the `oauth-login-site` website also with the authentication-service. It is required to pass to the website the query params `appId=APP_ID`.
So, if you have the configured app with id `my-awesome-app` in the authentication-service, and expose the login site under the url `http://my-host/login-site`, you should go to `http://my-host/login-site?app=my-awesome-app&redirect=/` to use the correct authentication service app and, at the end of the login, redirect to the correct path.

**Reply by malta895** [reply]
Hi @davidebianchi , thanks for your reply!

Are the query parameters proxied to the authentication service? I see that you mention only the `appId`, but the authentication service's authorize [requires the `providerId`](https://docs.mia-platform.eu/docs/runtime_suite/authentication-service/usage#authorize) as well.

Also, can I pass other parameters such as the `state`, and omit the `redirect` if my authentication service is configured with a `redirectUrlOnSuccessfulLogin` as a fallback redirect URI?

**Reply by davidebianchi** [reply]
I think yes, but I'm not sure. If you don't use the authorize page, you can use only the callback page which send to the server only code and state.
